mousebender/simple.py

```diff
@@ -45,34 +23,37 @@ def create_project_url(base_url: str, project_name: str) -> str:
     return "".join([base_url, packaging.utils.canonicalize_name(project_name), "/"])
 
 
-def _normalize_project_url(url):
-    """Normalizes a project URL found in a repository index.
+_Meta = TypedDict("_Meta", {"api-version": Literal["1.0"]})
 
-    If a repository is fully-compliant with PEP 503 this will be a no-op.
-    If a repository is reasonably compliant with PEP 503 then the resulting URL
-    will be usable.
 
-    """
-    url_no_trailing_slash = url.rstrip("/")  # Explicitly added back later.
-    base_url, _, project_name = url_no_trailing_slash.rpartition("/")
-    return create_project_url(base_url, project_name)
+class ProjectIndex(TypedDict):
```

Comment:
Thanks to [PEP 700](https://peps.python.org/pep-0700/), this will need to be versioned. Probably name it something like `ProjectIndex_1_0` or something and then have a variable for `ProjectIndex` that covers all possible versions. This goes for 

Don't have to change it in this PR, though; I'm just thinking out loud.